airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlCdcStateHandler.java

```diff
@@ -55,4 +57,22 @@ public AirbyteMessage saveStateAfterCompletionOfSnapshotOfNewStreams() {
     throw new RuntimeException("Snapshot of individual tables is not implemented in MSSQL");
   }
 
+  @Override
+  public boolean isSnapshotEvent(ChangeEvent<String, String> event){
```

Comment:
suggestion: implement as `default` interface method in `CdcStateHandler`

Reply:
Added the new functions as `default` so only `source-postgres` is overriding them 😄